dbt_semantic_interfaces/model_transformer.py

```diff
@@ -33,28 +21,10 @@ class SemanticManifestTransformer:
     Generally used to make it more convenient for the user to develop their model.
     """
 
-    PRIMARY_RULES: Sequence[SemanticManifestTransformRule] = (
```

Comment:
It would be nice to have some indication of why this change is useful or necessary in the commit message.